wgengine/userspace.go

```diff
@@ -133,6 +135,19 @@ type userspaceEngine struct {
 	// Lock ordering: magicsock.Conn.mu, wgLock, then mu.
 }
 
+type metrics struct {
+	// offline is the number of nodes magicsock attempted to connect to which are offline.
+	offline uint32
+	// relayed is the number of nodes magicsock connected thru over DERP.
```

Comment:
"number of nodes" isn't right on these.

this is about the number of new flows